src/main/kotlin/no/nav/tiltaksarrangor/service/DeltakerMapper.kt

```diff
@@ -186,9 +186,31 @@ fun DeltakerDbo.getAdresse(deltakerliste: DeltakerlisteDbo) = if (deltakerliste.
 	null
 }
 
-fun DeltakerDbo.getGjeldendeVurdering(): Vurdering? = vurderingerFraArrangor?.firstOrNull { it.gyldigTil == null }?.toVurdering()
+fun DeltakerDbo.getGjeldendeVurdering(): Vurdering? {
+	val gjeldendeVurdering = vurderingerFraArrangor?.maxByOrNull { it.opprettet } ?: return null
+	return Vurdering(
+		vurderingstype = gjeldendeVurdering.vurderingstype,
+		begrunnelse = gjeldendeVurdering.begrunnelse,
+		gyldigFra = gjeldendeVurdering.opprettet,
+		gyldigTil = null,
+	)
+}
+
+fun DeltakerDbo.getHistoriskeVurderinger(): List<Vurdering>? {
```

Comment:
Siden historiske vurderinger ikke er i bruk i frontend kunne du også vurdere å bare fjerne det derifra. Men kanskje det kommer senere.

Reply:
Kanskje det er bedre å fjerne historiske vurderinger..? Hvis vi skal vise tidligere vurderinger er det vel mer nærliggende å tro at de skal vises som en del av historikken? Jeg lurer på om jeg bare skal fjerne det. Jeg er jo allerede litt bekymret for at responsene inneholder mye greier, og her er det jo noe som ikke vi egentlig trenger å returnere :)

Reply:
Jeg ville helst ikke endre vurdering-formatet som brukes ut mot frontend, så derfor er det to formater nå (det som er definert i appen som brukes mot frontend og det fra amt-lib). Når vi begynne rå jobbe med kurstiltakene kan det jo hende vi ser at vi kan forenkle dette :)